tests/importer/test_importer.py

```diff
@@ -265,10 +259,10 @@ def test_shadowed_basename():
         assert os.path.isfile('tests/resources/importer/foo/some_mod.py')
 
         foo = importlib.import_module('foo')
-        assert foo.__file__.endswith('foo/__init__.hy')
+        assert os.path.basename(foo.__file__) == '__init__.hy'
         assert foo.ext == 'hy'
         some_mod = importlib.import_module('foo.some_mod')
-        assert some_mod.__file__.endswith('foo/some_mod.hy')
+        assert os.path.basename(some_mod.__file__) == 'some_mod.hy'
```

Comment:
It would be more concise and idiomatic to use `pathlib` for these checks. `Path(some_mode.__file__).name == 'some_mod.hy'` etc

Reply:
Aright. Might as well change all uses of `os.path` to `pathlib` in this file.